centreon/www/front_src/src/AppTokens/TokenListing/TokenListing.cypress.spec.tsx

```diff
@@ -0,0 +1,265 @@
+import { renderHook } from '@testing-library/react-hooks/dom';
```

Comment:
@Noha-ElAbrouki can you run this component test many times in the CI to make sure it isn't flaky?

Reply:
oookkk

Reply:
i launched it 4 times , and the test passed all 4 times